### PR TITLE
Fixing cert makefiles to support docs

### PR DESCRIPTION
**Pull request by nmittler**
The Makefiles have recently been changed so that the generated files for self-signed certs have a `selfSigned` prefix. This requires extra steps to rename the files before generating the `cacerts` secret, which assumes certain file names.

This breaks the Makefile into 2: one for self-signed certs and another for certs based on a root CA from an existing k8s cluster.

[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[x] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any changes that may affect Istio users.

### CODE REVIEW

File: tools/certs/Makefile [hunk 1]
```diff
@@ -1,278 +0,0 @@
-.SUFFIXES: .csr .pem .conf
```
Comment (myidpt) [inline tools/certs/Makefile:1]: Do you want to include the two .mk files in this file? Something like:
```
-include Makefile.k8s.mk
-include Makefile.selfsigned.mk
```

File: tools/certs/README.md [hunk 2]
```diff
@@ -1,27 +1,29 @@
...
-using self signed root certs and store them under $NAMESPACE directory.
+The table below describes the targets supported by both Makefiles.
 
-- `make <NAMESPACE>-certs-k8s`: this will generate intermediate certificates and sign certificates for a virtual machine
- connected to the namespace $NAMESPACE using serviceAccount $SERVICE_ACCOUNT using root cert from k8s cluster and store
- them under $NAMESPACE directory.
+Make Target | Makefile | Description
+------ | -------- | -----------
+`root-ca` | `Makefile.selfsigned.mk` | Generates a self-signed root CA key and certificate.
+`k8s-root-ca` | `Makefile.k8s.mk` | Fetches the Istio CA from the cluster.
```
Comment (myidpt) [inline tools/certs/README.md:13]: You mean fetch-root-ca, right?

**Review by myidpt — COMMENTED** [review]
Thanks for working on this! Generally LGTM.

**Review by myidpt — APPROVED** [review]

File: tools/certs/README.md (hunk 2 above)
Comment (nmittler) [reply tools/certs/README.md]: Yup ... fixed.

**Review by howardjohn — APPROVED** [review]

**Pull request merged**